manager-parent/manager-core/src/main/resources/db/migration/0.2/V0.2.1__create_v2_db.sql

```diff
@@ -30,7 +30,7 @@ CREATE TABLE PROCESSOR_V2 (
     id                             varchar(255) NOT NULL PRIMARY KEY,
 
     name                           varchar(255) NOT NULL,
-    flows                          jsonb        NOT NULL,
+    definition                     jsonb        NOT NULL,
```

Comment:
you need a new migration in order to apply this change

Reply:
Done.